introduction.md

```diff
@@ -2,7 +2,9 @@ Hi. I am José Cage. A web developer and product designer from Angola (African C
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?